int16 posStepper1
int16 posStepper2
int16 posStepper3
int16 posStepper4
int16 posStepper5
int16 posStepper6